test/functional/cypress/specs/contacts/activity-spec.js

```diff
@@ -54,19 +54,31 @@ describe('Contact activity', () => {
         )
       })
 
-      context('when viewing a Data Hub interaction', () => {
+      context('when viewing a Contact with Data Hub interaction', () => {
```

Comment:
```suggestion
      context('when viewing a contact with Data Hub interactions', () => {
```
I think this wording can be improved slightly